containers/unit_tests/TestOffsetView.hpp

```diff
@@ -127,7 +127,7 @@ void test_offsetview_construction(unsigned int size) {
   typedef typename range_type::point_type point_type;
 
   range_type rangePolicy2D(point_type{{ovmin0, ovmin1}},
-                           point_type{{ovend0, ovend1}});
+                           point_type{{ovend0, ovend1}}, point_type{{0, 0}});
```

Comment:
what is this point_type{{0,0}} here added for? Also doesn't this kind of explicitly assume that `{0,0}` means default for tiling?

Reply:
Yes, this is for tiling and the tiling argument is zero-initialized by default. Only `gcc-4.8.4` complains here with `-Wmissing-field-initializers`. If we don't want all of `-Wextra`, I would just drop this change.

Reply:
Yeah we don't do this, in particular not for gcc 4.8.4